Jenkinsfile

```diff
@@ -202,6 +202,51 @@ def get_macos_pipeline() {
   }  // return
 }  // def
 
+def get_win10_pipeline() {
+  return {
+    node('windows10') {
+      // Use custom location to avoid Win32 path length issues
+      ws('c:\\jenkins\\') {
+      cleanWs()
+      dir("${project}") {
+        stage("win10: Checkout") {
+          checkout scm
+        }  // stage
+
+        stage("win10: Conan setup") {
+          withCredentials([
+            string(
+              credentialsId: 'local-conan-server-password',
+              variable: 'CONAN_PASSWORD'
+            )
+          ]) {
+            bat """C:\\Users\\dmgroup\\AppData\\Local\\Programs\\Python\\Python36\\Scripts\\conan.exe user \
+              --password ${CONAN_PASSWORD} \
+              --remote ${conan_remote} \
+              ${conan_user}"""
+          }  // withCredentials
+        }  // stage
```

Comment:
This step isn't strictly required if we're not pushing to the local Conan server, but let's leave this here in case we want to do it in the future.